mocks/mocks.py

```diff
@@ -184,7 +213,8 @@ def create_mock_galaxy_noiseless_image(config,
     Parameters
     ----------
     config : dict
-        a dictionary consist of configuration data for the function
+        a dictionary consist of configuration data for the function, usually
+        the first output of the `mocks.prepare_mocks` function.
```

Comment:
Usually? Seems like you have to call `prepare_mocks` on the `config` before calling this otherwise it won't work.